### PR TITLE
Cases panel coming soon state

### PR DESCRIPTION
**Pull request by JordanSh**
## Summary
solves:
- https://github.com/elastic/security-team/issues/2759

needs to be merged after:
- https://github.com/build-security/kibana/pull/118

### CODE REVIEW

File: x-pack/plugins/cloud_security_posture/public/common/utils/getFormattedNum.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getFormattedNum = (num: number) =>
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/common/utils/getFormattedNum.ts:8]: 1. cool !
2. filename should be `snake_case`

File: x-pack/plugins/cloud_security_posture/public/components/chart_panel.tsx [hunk 2]
```diff
@@ -61,13 +61,16 @@ export const ChartPanel: React.FC<ChartPanelProps> = ({
...
   return (
     <EuiPanel hasBorder={hasBorder} hasShadow={false} data-test-subj="chart-panel">
-      {title && (
-        <EuiTitle size="s" css={euiTitleStyle}>
-          <h3>{title}</h3>
-        </EuiTitle>
-      )}
-      <EuiSpacer />
-      {renderChart()}
+      <EuiFlexGroup direction="column" gutterSize="none" style={{ height: '100%' }}>
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/components/chart_panel.tsx:64]: do we need the `height` addition here?

File: x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    [history]
+  );
+
+  const handleViewAllClick = useCallback(
+    () =>
+      history.push({
+        pathname: allNavigationItems.findings.path,
+        search: encodeQuery(failedFindingsQuery),
+      }),
+    [history]
```
Comment (orouz) [inline x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx:108]: not sure if `history` is stable
i _think_ `push` is tho

**Comment by ari-aviran** [comment]
> # TO BE REVIEWED AFTER #118 IS MERGED

This is easily missed, please don't assign reviewers until a PR is ready for review :)

**Review by ari-aviran — CHANGES_REQUESTED** [review]
Please re-request a review after #118 is merged

**Comment by JordanSh** [comment]
> > # TO BE REVIEWED AFTER #118 IS MERGED
> 
> This is easily missed, please don't assign reviewers until a PR is ready for review :)

of course. not opening as draft was a miss click

**Comment by ari-aviran** [comment]
@JordanSh please rebase this over `main` (you can use the cherry-picking method @orouz suggested)

**Pull request closed without merging**